docs/coins_spends_and_wallets.md

```diff
@@ -239,6 +239,8 @@ Before we go there, let's explain signatures:
 
 ## BLS Aggregated Signatures
 
+If you don't have a fundamental understanding of cryptographic signatures, it will be good to [take a look at that](https://en.wikipedia.org/wiki/Digital_signature) before you continue.
```

Comment:
how about "it will be good to familiarize yourself with [digital signatures] ... before you continue."

Reply:
How does it look now?

Reply:
"Take a look at that" is generally considered a bad link since you are linking to a text on "digital signatures"